### DISCUSSION TITLE
Swipe seeking

### DISCUSSION THREAD

**Discussion by Famienn** [kingslay/KSPlayer#814] [Ideas]
Hey! Great player, works superbly. I have a suggestion that I would really appreciate - a feature similar to Outplayer, where you can seek by swiping the screen left and right in addition to the normal seeking, thanks!

**Comment by kingslay** [comment]
你是指iOS还是macOS呢？tvOS现在应该就支持 swipe seeking了吧。

**Reply by Famienn** [reply]
yeah i meant iOS

**Comment by kingslay** [comment]
我增加了这个功能了。你用最新的testflight包试下。
